beacon-chain/db/kv/lightclient.go

```diff
@@ -297,6 +310,8 @@ func keyForLightClientUpdate(v int) ([]byte, error) {
 		return denebKey, nil
 	case version.Capella:
 		return capellaKey, nil
+	case version.Bellatrix:
+		return bellatrixKey, nil
 	case version.Altair:
 		return altairKey, nil
```

Comment:
```suggestion
	case version.Altair, version.Bellatrix:
		return altairKey, nil
```
This would also work, right? Not sure which one I like better

Reply:
I think separating them is cleaner. but I'm not sure either.